review-tools/gitaddrev

```diff
@@ -107,8 +112,13 @@ foreach (@ARGV) {
 	$verbose = 1;
     } elsif (/^--web$/) {
         $WHAT = 'web';
+        $min_reviewers = 1;
+        # openssl/web is governed by OMC
+        $min_otc = 0;
+        $min_omc = 1;
     } elsif (/--tools$/) {
-        $WHAT = 'tools'
+        $WHAT = 'tools';
+        $min_reviewers = 1;
```

Comment:
We could do without this line, if the OTC feels that changes to the tools need two reviewers

Reply:
Personally, I'm okay with one reviewer.
We use these tools, we'll find out soon enough if they break :grinning: